### PR TITLE
Electrician code

### PR DESCRIPTION
**Pull request by SquawkyArctangent**
shocking

#2089

### CODE REVIEW

**Comment by Haganeko** [comment]
What did the new commit change?

**Comment by Haganeko** [comment]
A hero with no inventory, lol

**Comment by chrisinajar** [comment]
master has that gold.js fix if you merge it in

File: game/scripts/npc/abilities/electrician_talents_oaa.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		}
+	}
+	//=================================================================================================================
+	// Ability: Special Bonus
+	//=================================================================================================================
+	"special_bonus_hp_1000"
+	{
+		// General
+		//-------------------------------------------------------------------------------------------------------------
+		"BaseClass"				"special_bonus_hp_100"
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/electrician_talents_oaa.txt]: That works? waaaaaaaow

**Comment by chrisinajar** [comment]
This is essentially done, i want to wait for model/animations/icons so it looks good.

**Comment by Trildar** [comment]
I'd be a little surprised if Atan didn't notice this herself, but just to note: while Baumi was testing on stream, it seemed like the channel time on the Q doesn't interact well with status resistance as it doesn't adjust to the actual time the stun applies. I believe Shackles does do that, though I haven't actually tested it. I mean, it could be considered a feature since the ability has a secondary effect aside from the stun, but it still feels kinda weird.

**Comment by SquawkyArctangent** [comment]
obtained a cool almost $60 hell yeah

**Pull request merged**